File: app/src/main/java/org/test/kotlin/demo/users/UsersModule.java
package org.test.kotlin.demo.users;

import dagger.Binds;
import dagger.Module;

@Module
public interface UsersModule {

    @Binds
    UsersContract.View bindView(UsersFragment impl);

    @Binds
    UsersContract.Presenter bindPresenter(UsersPresenter impl);

}

File: app/src/main/java/org/test/kotlin/demo/users/UsersModule.kt
package org.test.kotlin.demo.users

import dagger.Binds
import dagger.Module

@Module
interface UsersModule {

    @Binds
    fun bindView(impl: UsersFragment): UsersContract.View

    @Binds
    fun bindPresenter(impl: UsersPresenter): UsersContract.Presenter

}
